gradle/config/checkstyle/h2o_checks.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!--
+
+    Checkstyle configuration that checks the Google coding conventions from:
+    
+    -  Google Java Style
+       https://google-styleguide.googlecode.com/svn-history/r130/trunk/javaguide.html
+       
+    Checkstyle is very configurable. Be sure to read the documentation at
+    http://checkstyle.sf.net (or in your downloaded distribution).
+
+    Most Checks are configurable, be sure to consult the documentation.
+
+    To completely disable a check, just comment it out or delete it from the file.
+
+    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
+    
+    This file is specific for H2O.ai. We plan to grow it gradually.
+ -->
+
+<module name = "Checker">
+  <module name="TreeWalker">
+    <module name="Regexp">
+        <property name="format" value="[^)]\.getBytes\("/>
```

Comment:
I'll add custom message; regarding using only string literals, it probably won't work; there's a lot of cases where a string is in a variable. I don't see a way to limit to specific data types - it's all on a macro level.